docs/HowToPortTizenRTHeaderofBinary.md

```diff
@@ -0,0 +1,50 @@
+## Guide to binary header
```

Comment:
1. Please remove one # from all of titles.
2. Let's use the same title with file name.